app/controllers/api/base_controller.rb

```diff
@@ -1,7 +1,7 @@
 class API::BaseController < ActionController::Base
   include CurrentUserHelper
   include ::ProtectedFromForgery
-  before_filter :require_authenticated_user
+  before_filter :require_authenticated_user, except: :index
```

Comment:
I'm not ready to fully commit to making the API public-accessible, but I think the index action should be a good place to start.

Next steps will be to go through groups, discussions, maybe comments controllers, and add the `visible_to_public` methods to them.